contracts/test/PolygonMocks.sol

```diff
@@ -15,3 +17,29 @@ contract FxStateSenderMock {
     // solhint-disable-next-line no-empty-blocks
     function sendMessageToChild(address _receiver, bytes calldata _data) external {}
 }
+
+contract DepositManagerMock {
+    function depositERC20ForUser(
+        address token,
+        address user,
+        uint256 amount // solhint-disable-next-line no-empty-blocks
+    ) external {} // solhint-disable-line no-empty-blocks
+}
+
+contract PolygonRegistryMock {
+    // solhint-disable-next-line no-empty-blocks
+    function erc20Predicate() external returns (address predicate) {}
```

Comment:
I know this is a mock, but it looks like using the "registry" from [polygon's list](https://static.matic.network/network/mainnet/v1/index.json) may give the wrong value. The Registry is [0x33a02E6cC863D393d6Bf231B697b82F6e499cA71](https://etherscan.io/address/0x33a02E6cC863D393d6Bf231B697b82F6e499cA71#readContract), and you can see that function `erc20Predicate` returns [0x158d5fa3Ef8e4dDA8a5367deCF76b94E7efFCe95](https://etherscan.io/address/0x158d5fa3ef8e4dda8a5367decf76b94e7effce95). Just based on the tx's i see calling that contract, it's for exiting from Matic, but not bridging/depositing onto matic. 

Again since this is a mock, you can have it return whatever value you want. So not a problem. But it might lead to someone thinking that the actual mainnet contract will return the "deposit" predicate, which it doesn't. I believe the deposit predicate is [0x40ec5b33f54e0e8a33a975908c5ba1c14e5bbbdf](https://etherscan.io/address/0x40ec5b33f54e0e8a33a975908c5ba1c14e5bbbdf).

I'm thinking that Polygon_Adapter.sol is deployed on ETH mainnet, for transferring tokens over to Matic. If that's not true... this doesn't apply